packages/cozy-pouch-link/src/PouchManager.js

```diff
@@ -263,6 +281,54 @@ class PouchManager {
   getDatabaseName(doctype) {
     return `${this.options.prefix}_${doctype}`
   }
+
+  async warmupQueries(doctype, queries) {
+    if (!this.warmedUpQueries[doctype]) this.warmedUpQueries[doctype] = []
+    await Promise.all(
+      queries.map(query => {
+        const def = getQueryAlias(query)
+        if (!this.warmedUpQueries[doctype].includes(def)) {
+          this.warmedUpQueries[doctype].push(def)
+          try {
+            return this.executeQuery(query.definition().toDefinition())
```

Comment:
I think we want to await the end of the executeQuery. Otherwise, in case of async error, we will not delete the entry in this.warmedUpQueries.

Reply:
Hmm, I think I've this commit somewhere, let me find it 👍